e2e-tests/functions

```diff
@@ -240,8 +240,8 @@ wait_for_delete() {
 deploy_pmm_server() {
 	local platform=kubernetes
 	helm uninstall -n "${NAMESPACE}" pmm || :
-	helm install monitoring -n "${NAMESPACE}" --set imageTag=$IMAGE_PMM_SERVER_TAG --set service.type="LoadBalancer" \
-		--set imageRepo=$IMAGE_PMM_SERVER_REPO --set platform="${platform}" "https://percona-charts.storage.googleapis.com/pmm-server-${PMM_SERVER_VERSION}.tgz"
+	helm install monitoring -n "${NAMESPACE}" --set imageTag=${IMAGE_PMM_SERVER#*:} --set service.type="LoadBalancer" \
+		--set imageRepo=${IMAGE_PMM_SERVER%:*} --set platform="${platform}" "https://percona-charts.storage.googleapis.com/pmm-server-${PMM_SERVER_VERSION}.tgz"
```

Comment:
It looks nice, but potentially can confuse and make debug more difficult. 
@tplavcic what do you think?

Reply:
@nmarukovich  all the other images we have only one parameter for both

Reply:
I think it should be ok, since we only assign it here and nowhere else and it's a very straightforward assignment. Also if incorrect image gets parsed pmm deployment will just fail to run so it shouldn't be something very complicated to debug.